knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml

```diff
@@ -962,6 +962,21 @@ spec:
           type: Utilization
           averageUtilization: 100
 ---
+# Webhook PDB.
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: kafka-webhook
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+spec:
+  minAvailable: 80%
```

Comment:
I think we want this to be `1`. Instead of percentage /cc @maschmid

Reply:
yep -- we chatted about this, but at the same time I was under the impression we wanted to get that upstream first?

The alternative is that we can create another patch to apply changing it to `1` in the interm, is that the approach you'd prefer for now?

Reply:
@matzew I've submitted pdb's with absolute values (`1`) for eventing-kafka and eventing in knative-sandbox/eventing-kafka#703 and knative/eventing#5494

Reply:
very cool!

Reply:
> The alternative is that we can create another patch to apply changing it to 1 in the interim, is that the approach you'd prefer for now?

yep, let's do that afterwards (for eventing we have set it here already to `1`)